static/app/views/starfish/components/sampleEvents.tsx

```diff
@@ -0,0 +1,100 @@
+import Duration from 'sentry/components/duration';
+import GridEditable, {GridColumnHeader} from 'sentry/components/gridEditable';
+import Link from 'sentry/components/links/link';
+import EventView from 'sentry/utils/discover/eventView';
+import {
+  DiscoverQueryProps,
+  useGenericDiscoverQuery,
+} from 'sentry/utils/discover/genericDiscoverQuery';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {TextAlignLeft} from 'sentry/views/starfish/modules/APIModule/endpointTable';
+
+type Props = {
+  eventView: EventView;
+};
+
+type DataRow = {
+  id: string;
+  'transaction.duration': number;
+};
+
+const COLUMN_ORDER = [
+  {
+    key: 'id',
+    name: 'Event ID',
+    width: 300,
+  },
+  {
+    key: 'transaction.duration',
+    name: 'Duration',
+    width: -1,
+  },
+];
+
+export function SampleEvents({eventView}: Props) {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const sampleEventsEventView = eventView
+    .clone()
+    .withColumns([
+      {
+        field: 'transaction.duration',
+        kind: 'field',
+      },
+    ])
+    .withSorts([
+      {
+        field: 'transaction.duration',
+        kind: 'desc',
+      },
+    ]);
+
+  function renderBodyCell(column: GridColumnHeader, row: DataRow): React.ReactNode {
```

Comment:
If you go with the previous suggestion
```suggestion
  function renderBodyCell(column: TableColumnHeader, row: DataRow): React.ReactNode {
```